erpnext_com/www/docs/user/manual/en/manufacturing/introduction.md

```diff
@@ -48,6 +48,6 @@ with the Project module.
 You can track work-in-progress by creating work-in-progress Warehouses.
 
 ERPNext will help you track material movement by automatically creating Stock
```

Comment:
unsure about this change, the meaning changes... need to confirm which one is right